src/app/layout/NavigationDrawer/NavigationDrawer.style.js

```diff
@@ -25,3 +25,7 @@ export const useStyles = makeStyles({
     color: "#150E60",
   },
 });
+
+export const iconStyle = {
```

Comment:
this should go into the useStyles

Reply:
Sure @mat10tng , I was looking at material-ui documentation and they haven't specified a use case with useStyles. https://material-ui.com/components/icons/ 
I thought instead of writing it inline I could offload it to the style.js. What are your thoughts?

Reply:
This is because of consistency tbh, just so that it would make it easier for other to read